### PR TITLE
Avoiding race-conditions on deleting test data on DBFS

### PR DESCRIPTION
**Pull request by falaki**
Randomizing dbfs path for test data.

### CODE REVIEW

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2438-1/167c54e2-9907-4476-8812-f19bfe46d909.txt).

**Comment by falaki** [comment]
Databricks Connect tests failed. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2438-1/167c54e2-9907-4476-8812-f19bfe46d909.txt).

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2438-2/47ba7717-8ec2-49fa-86c1-e002045be9c4.txt).

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2438-3/b2b5d3b0-4a83-421f-861e-5d805fb455b9.txt).

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2438-4/5fffc926-7e92-498d-ba29-bebf1d35f728.txt).

File: Jenkinsfile [hunk 1]
```diff
@@ -57,23 +58,25 @@ pipeline {
...
         }
     }
     post {
         always {
             bash "databricks clusters delete --cluster-id ${clusterId} 2>&1 | tee -a log.txt "
-            bash """dbfs rm -r dbfs:/tmp/data  2>&1 | tee -a log.txt"""
+            bash """dbfs rm -r ${dbfsPath}  2>&1 | tee -a log.txt"""
             s3Upload(file: 'log.txt', bucket:'sparklyr-jenkins', path: s3Path, contentType: 'text/plain; charset=utf-8')
+        }
+        success {
```
Comment (Loquats) [inline Jenkinsfile:79]: Nice, seems like this should fix the success+failure message too.

**Pull request merged**